src/features/program/api/useGetProgram.ts

```diff
@@ -0,0 +1,31 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import { useApiClient } from '~/providers/ApiClientProvider'
+import { ProgramResponse } from '../types/program'
+
+export const useGetProgram = (
+  params: { programId: number },
+  options?: UseQueryOptions<
+    unknown,
+    unknown,
+    ProgramResponse,
+    (number | string)[]
+  >
+) => {
+  const { programId } = params
+
+  const { axios, api } = useApiClient()
+
+  return useQuery(
+    ['online-course', programId],
```

Comment:
```suggestion
    ['online-course', programId],
```
key nya online-course?

Reply:
astaga okeehh, efek copas nih gw format nya wkwk